make/RunTests.gmk

```diff
@@ -996,7 +996,8 @@ define SetupRunJtregTestBody
 	$$(RM) -r $$($1_TEST_RESULTS_DIR)
 
   $1_COMMAND_LINE := \
-      $$(JTREG_JAVA) $$($1_JTREG_LAUNCHER_OPTIONS) \
+      cd $$($1_TEST_SUPPORT_DIR) && $$(JTREG_JAVA) \
+      $$($1_JTREG_LAUNCHER_OPTIONS) \
```

Comment:
This this be aligned with the row below?

Reply:
I didn't notice that. Actually, I think it is the remaining lines that are incorrectly indented.

Reply:
With some more context, I see that the original author tried to separate the jtreg arguments from the additional structure needing to execute things properly in make. I made this more explicit by extracting the arguments to a separate variable, which led to more natural indentation.